docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,151 @@
+# How to integrate Autoware with your vehicle
+
+This page demonstrates how to integrate Autoware with a real vehicle.
+
+## 1. Prepare your real vehicle hardware
+
+Prerequisites for the vehicle:
+
+- car-like or diff-drive vehicle
```

Comment:
Probably you should capitalize each item?

https://developers.google.com/style/lists?hl=en
```suggestion
- Car-like or diff-drive vehicle
```